<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>schedule::AdvancedSettingsDialog</class>
 <widget class="QDialog" name="schedule::AdvancedSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notification Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="preview_btn">
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="timeout_edit">
     <property name="suffix">
      <string extracomment="seconds"> sec</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="show_tray_icon">
     <property name="text">
      <string>Show additional tray icon for plugin</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="browse_btn">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="msg_balloon_rbtn">
     <property name="text">
      <string>&amp;Balloon message</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>89</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="msg_dialog_rbtn">
     <property name="text">
      <string>Message &amp;dialog</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="play_sound_enabled">
     <property name="text">
      <string>&amp;Play sound</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QRadioButton" name="msg_not_show_rbtn">
     <property name="text">
      <string>Do not show notification</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="timeout_label">
     <property name="text">
      <string>&amp;Timeout:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>timeout_edit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QKeySequenceEdit" name="add_task_seq_edit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="hotkey_add_task_enabled">
     <property name="text">
      <string>Shortcut for quick task addition</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>schedule::AdvancedSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>schedule::AdvancedSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hotkey_add_task_enabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>add_task_seq_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
